fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -620,7 +617,10 @@ ACTOR Future<GetReadVersionReply> getLiveCommittedVersion(SpanContext parentSpan
 	// before the request returns, so it is committed. (2) No proxy on our list reported committed a higher version
 	// before this request was received, because then its committedVersion would have been higher,
 	//     and no other proxy could have already committed anything without first ending the epoch
-	state Span span("GP:getLiveCommittedVersion"_loc, parentSpan);
+	state Span span("GP:getLiveCommittedVersion"_loc);
+	for (const SpanContext& spanContext : spanContexts) {
+		span.addLink(spanContext);
+	}
```

Comment:
I think you can use `span.addLinks(spanContexts)` here.

Reply:
I tried that, but couldn't get that to work: https://stackoverflow.com/questions/18895583/convert-a-vectort-to-initializer-listt

Reply:
The C++ way would probably be to accept iterators